ipn/ipnlocal/local_test.go

```diff
@@ -457,7 +458,7 @@ func TestLazyMachineKeyGeneration(t *testing.T) {
 	panicOnMachineKeyGeneration = true
 
 	var logf logger.Logf = logger.Discard
-	store := new(ipn.MemoryStore)
+	store := new(mem.Store)
```

Comment:
in all these other places we use "store" as the variable name, so it's probably fine to use in tailscaled_windows.go even if you shadow the package name for a few lines

Reply:
I can't do that because we need to try `store.NewFileStore` if `store.New` fails with `ErrNoMatchingStore`